src/main/java/ee/tuleva/onboarding/mandate/email/MandateEmailSender.java

```diff
@@ -27,7 +31,24 @@ public void sendEmail(AfterMandateSignedEvent event) {
     PaymentRates paymentRates = paymentRateService.getPaymentRates(event.getUser());
     PillarSuggestion pillarSuggestion =
         new PillarSuggestion(event.getUser(), contactDetails, conversion, paymentRates);
-    emailService.sendMandate(
-        event.getUser(), event.getMandate(), pillarSuggestion, event.getLocale());
+    if (!event.isPartOfBatch()) {
+      mandateEmailService.sendMandate(
+          event.getUser(), event.getMandate(), pillarSuggestion, event.getLocale());
+    } else {
+      log.info(
+          "Skipping mandate email for (id:{}) because it is part of batch",
```

Comment:
Prefer to use the following pattern for logging:
"Skipping mandate email because it is part of a batch: mandateId={}"